components/Players/PlayerManager.vue

```diff
@@ -193,13 +200,45 @@ export default Vue.extend({
         default:
           return this.getCurrentItem.Name;
       }
+    },
+    timeLeft(): number {
+      return parseInt((this.mediaDuration - this.getCurrentTime).toFixed());
+    },
+    mediaDuration(): number {
+      // In seconds
+      return this.ticksToMs(this.getCurrentItem.RunTimeTicks) / 1000;
+    },
+    showUpNext(): boolean {
+      if (
+        this.isMinimized ||
+        this.upNextUserHidden ||
+        this.getCurrentItem.Type !== 'Episode' ||
+        !this.getNextItem
+      ) {
+        return false;
+      }
+      // How much of the video to go through before showing
+      // the 'up-next' component
+      const showAtProgressPercentage = 0.97;
+
+      if (
+        this.getCurrentTime <=
+        this.mediaDuration * showAtProgressPercentage
+      ) {
+        return false;
+      }
+      return true;
     }
   },
   created() {
     this.$store.subscribe((mutation, state: AppState) => {
       switch (mutation.type) {
         case 'playbackManager/INCREASE_QUEUE_INDEX':
+          this.upNextUserHidden = false;
+          break;
         case 'playbackManager/DECREASE_QUEUE_INDEX':
+          this.upNextUserHidden = false;
+          break;
```

Comment:
I don't think we should break here?

Reply:
Wouldn't it fall through if not? ESLint complains if i remove the break: `Expected a 'break' statement before 'case'.`

Reply:
Haven't gotten any comments on this. Marking as resolved.

Reply:
It's definitely not the same logic as before the PR though. Currently both `INCREASE_QUEUE_INDEX` and `DECREASE_QUEUE_INDEX` finally executed the code of the case `SET_CURRENT_ITEM_INDEX`, and now it won't. And yeah, it's good practice to keep the break when you have logic in your cases. So I'd say the best thing is to factorize somewhere the code of `SET_CURRENT_ITEM_INDEX` and call it in those 3 cases. :)

Reply:
This indeed breaks playback reporting when changing tracks.